osquery/tables/networking/CMakeLists.txt

```diff
@@ -116,6 +116,7 @@ function(generateOsqueryTablesNetworking)
   elseif(DEFINED PLATFORM_WINDOWS)
     list(APPEND public_header_files
       windows/interfaces.h
+      windows/windows_firewall_rules.h
```

Comment:
@alessandrogario don't really like that I had to expose this header for requested testability.
let me know, what you think. can rollback this and the previous commit.

Reply:
I just prefer not to expose the implementation details in the public headers for testability sake only.